src/ui/Misc/Slider.ts

```diff
@@ -756,15 +756,15 @@ class SliderGraph {
   }
 
   private padGraphWithEmptyData(data: ISliderGraphData[]) {
-    let oneStepOfGraph = data[0].end - data[0].start;
+    const oneStepOfGraph = data[0].end - data[0].start;
     this.padBeginningOfGraphWithEmptyData(data, oneStepOfGraph);
     this.padEndOfGraphWithEmptyData(data, oneStepOfGraph);
   }
 
   private padBeginningOfGraphWithEmptyData(data: ISliderGraphData[], oneStepOfGraph: number) {
     if (data[0].start > this.slider.options.start && data[0].start > oneStepOfGraph) {
       const difToFillAtStart = data[0].start - this.slider.options.start;
-      const nbOfStepsAtStart = Math.round(difToFillAtStart / oneStepOfGraph);
+      const nbOfStepsAtStart = Math.round(difToFillAtStart / (oneStepOfGraph || 1));
```

Comment:
Rather than adding `|| 1` here and on line 780 to ensure the `oneStepOfGraph` is a good value, it would be better to do it on line 759. I would even suggest extracting that line into a separate method:

```
private calculateOneStepOfGraph(data: ISliderGraphData[]) {
    const oneStep = data[0].start - this.slider.options.start; // possibly wrapping this with Math.abs if it is needed.
    return oneStep || 1;
}
```